Examples/Sync-UiPathADUsers.ps1

```diff
@@ -42,12 +46,14 @@ param(
     [Parameter(Mandatory=$true, Position=1)] 
     [HashTable] $RolesMapping,
     [Parameter(Mandatory=$false)]
-    [ValidateSet('Remove', 'List', 'Ignore')]
+    [ValidateSet('Remove', 'List', 'Ignore', 'Lock')]
     [string] $OrphanedUsersAction = 'Ignore',
     [Parameter()]
     [switch] $AllowUsernameTruncate,
     [Parameter()]
-    [string[]] $AllowedDomainNames = @()
+    [string[]] $AllowedDomainNames = @(),
+    [Parameter()]
```

Comment:
`$AllowedDomainNames` and `$AllosAllDomainNames` should each be in a parameter set, so that they cannot be both provided simultaneously,

Reply:
done!